articles/application-gateway/configuration-infrastructure.md

```diff
@@ -24,7 +24,7 @@ An application gateway is a dedicated deployment in your virtual network. Within
 
 Application Gateway uses one private IP address per instance, plus another private IP address if a private front-end IP is configured.
 
-Azure also reserves five IP addresses in each subnet for internal use: the first four and the last IP addresses. For example, consider 15 application gateway instances with no private front-end IP. You need at least 20 IP addresses for this subnet: five for internal use and 15 for the application gateway instances.
+Azure also reserves [five IP addresses in each subnet for internal use: the first four and the last IP addresses](../virtual-network/virtual-networks-faq.md#are-there-any-restrictions-on-using-ip-addresses-within-these-subnets). For example, consider 15 application gateway instances with no private front-end IP. You need at least 20 IP addresses for this subnet: five for internal use and 15 for the application gateway instances.
```

Comment:
Please notify me if this related path is wrong.
I'm not familiar with how links work in the docs.